_posts/2022-07-15-documentation-framework.md

```diff
@@ -0,0 +1,90 @@
+
+We want the docs to quickly answer questions like:
+
+ - I know my workflow is parallizable - can Dask help?
+ - How do I find my logs for a particular worker?
+ - Should I be using `.persist()`?
+
+## Contents
+
+* [Theory](#theory)
+* [Current Documentation](#current-documentation)
+* [New Structure](#new-structure)
+* [How you can help](#how-you-can-help)
+
+## Theory
+
+The Diataxis Framework proposes that the documentation be split into 4 entirely separate sections.
+
+![Diataxis Framework](/images/diataxis-framework.png)
+_Credit: https://diataxis.fr/_
+
+Each section serves a unique purpose.
+
+* **Tutorials** provide a narrative that addresses a particular larger objective such as predicting global temperature or analyzing financial data.
+* **How-Tos** target people who already know _what_ they want to do and are trying to figure out _how_ to do it. These people might ask questions like:
+  - How do I apply a rolling mean to a timeseries?
+  - How do I groupby a column?
+  - How do I write to a geotiff?
+* **Reference** provides the exact arguments and outputs of a particular operation.
+* **Explanation** gives context and includes descriptions of how operations work internally.
+
+## Current Documentation
+
+There are several different sites that comprise different aspects of dask documentation. Of particular interest are [Examples](https://examples.dask.org), [Tutorials](https://tutorial.dask.org) and [Docs](https://docs.dask.org).
+
+The bulk of the documentation that we currently have on [Docs](https://docs.dask.org) falls under "Explanation" and "Reference" but they are pretty intermingled. There are also some small "How-Tos" sprinkled in, particularly in the API docs.
+
+The material on [Tutorials](https://tutorial.dask.org) is a mixture of "Tutorial" and "Explanation". They answer questions like: "What can I do with Dask Dataframes?" *and* questions like "What is a Dask Dataframe?". These are styled like lectures in that there is often no motivating example and the assumption is that the audience wants to learn both about how to do specific operations in dask and how those operations work.This type of material can be consumed as standalone content and runs on binder.
+
+[Examples](https://examples.dask.org) pretty much falls under "How-To" but there is a fair amount of setup and each example isn't split into small enough bits. They answer questions like: "How do I use dask dataframes?" and they have some more longer workflows.
+
+## New Structure
+
+[Tutorial](https://tutorial.dask.org) will be left as is and treated as a long-form overview.
+
+[Examples](https://examples.dask.org) will be presented more as **How-Tos** with little explanation and more code. Historically one of the current roles of examples is to demonstrate what Dask looks like, that role is now subsumed by "10 Minutes to Dask". 
```

Comment:
```suggestion
[Examples](https://examples.dask.org) will be presented more as **How-Tos** with little explanation and more code. This will be similar to gallery or cookbook style documentation that you may see in other projects. Historically one of the current roles of examples is to demonstrate what Dask looks like, that role is now subsumed by "10 Minutes to Dask". 
```